libs/books/feature/src/lib/books-feature.module.ts

```diff
@@ -28,6 +29,7 @@ const EXPORTS = [
     MatButtonModule,
     MatBadgeModule,
     MatIconModule,
+    MatTooltipModule,
```

Comment:
Is this really the best way to indicate the usage of the checkbox? It doesn't seem very accessible.

Reply:
you're right, not everyone will hover over the checkbox. Will make changes to let the user know what the checkbox is for in a better way.